scripts/generate_dump

```diff
@@ -859,24 +902,28 @@ save_log_files() {
 ###############################################################################
 # Save warmboot files
 # Globals:
-#  TARDIR, TARFILE, TAR, DUMPDIR, TECHSUPPORT_TIME_INFO
+#  TARDIR, TARFILE, TAR, DUMPDIR, TECHSUPPORT_TIME_INFO, NOOP
 # Arguments:
 #  None
 # Returns:
 #  None
 ###############################################################################
 save_warmboot_files() {
     # Copy the warmboot files
-    mkdir -p $TARDIR
-    $CP $V -rf /host/warmboot $TARDIR
-
-    start_t=$(date +%s%3N)
-    ($TAR $V --warning=no-file-removed  -rhf $TARFILE -C $DUMPDIR --mode=+rw \
-        $BASE/warmboot \
-        || abort "${ERROR_TAR_FAILED}" "Tar append operation failed. Aborting for safety.") \
-        && $RM $V -rf $TARDIR
-    end_t=$(date +%s%3N)
-    echo "[ Warm-boot Files ] : $(($end_t-$start_t)) msec" >> $TECHSUPPORT_TIME_INFO
+    if $NOOP; then
+        echo "$CP $V -rf /host/warmboot $TARDIR"
+    else
+        mkdir -p $TARDIR
+        $CP $V -rf /host/warmboot $TARDIR
+
+        start_t=$(date +%s%3N)
+        ($TAR $V --warning=no-file-removed  -rhf $TARFILE -C $DUMPDIR --mode=+rw \
+            $BASE/warmboot \
+            || abort "${ERROR_TAR_FAILED}" "Tar append operation failed. Aborting for safety.") \
+            && $RM $V -rf $TARDIR
+        end_t=$(date +%s%3N)
+        echo "[ Warm-boot Files ] : $(($end_t-$start_t)) msec" >> $TECHSUPPORT_TIME_INFO
```

Comment:
Why isn't the time info saved for only this case?

Reply:
Updated. Thanks for catching.